zerver/test_bugdown.py

```diff
@@ -175,6 +175,11 @@ def replaced(payload, url, phrase=''):
             converted = bugdown_convert(inline_url)
             self.assertEqual(match, converted)
 
+    def test_inline_file(self):
+        msg = 'Check out this file file:///Volumes/myserver/Users/Shared/pi.py'
+        converted = bugdown_convert(msg)
+        self.assertEqual(converted, '<p>Check out this file <a href="file:///Volumes/myserver/Users/Shared/pi.py" target="_blank" title="file:///Volumes/myserver/Users/Shared/pi.py">file:///Volumes/myserver/Users/Shared/pi.py</a></p>')
+
```

Comment:
Sorry for the slow reply!
I have try overriding settings. However it didn't seems to work.
The function sill follow the setting that I set in zproject/test_settings.
How can I solve this problem? Thanks
```
def test_inline_file(self):
    # Using initial settings I setup in test_settings (ENABLE_FILE_LINKS=False)
    msg = 'Check out this file file:///Volumes/myserver/Users/Shared/pi.py'
    converted = bugdown_convert(msg)
    self.assertEqual(converted, '<p>Check out this file file:///Volumes/myserver/Users/Shared/pi.py</p>') # Success
    # Test for Enabling file links
    with self.settings(ENABLE_FILE_LINKS=True):
        converted = bugdown_convert(msg)
        self.assertEqual(converted, '<p>Check out this file <a href="file:///Volumes/myserver/Users/Shared/pi.py" target="_blank" title="file:///Volumes/myserver/Users/Shared/pi.py">file:///Volumes/myserver/Users/Shared/pi.py</a></p>') # Failed
```